windows/uninstall.ps1

```diff
@@ -48,8 +55,14 @@ if ($ff.ToUpper() -eq 'Y') {
     Remove-Item $_ -Force -Recurse
     }
 
+# Close ports on network
```

Comment:
@aberdean awesome - maybe we have to ask here too if they want to close the ports again